### PR TITLE
Added sigmoid layer

### PR DESCRIPTION
**Pull request by AdityaKane2001**
@qlzh727 @fchollet 

Added Sigmoid activation layer for completeness. Not sure if we need masking here, and will the same method work as done in Softmax will work for Sigmoid. Please let me know your thoughts on the same.

### CODE REVIEW

**Comment by fchollet** [comment]
Thanks for the PR. As a general rule such an activation does not need to be a layer. You can just use `keras.activations.sigmoid`.

We only create a layer for an activation if:

- It is sufficient popular (this one qualifies) AND
- It has some element of trainable state
- OR It has constructor arguments

This one is purely stateless and non-configurable so it does not need to be a layer.

**Comment by AdityaKane2001** [comment]
@fchollet 

Thanks for the clarification.
